qiskit/extensions/standard/cu3.py

```diff
@@ -65,22 +65,21 @@ def inverse(self):
 def cu3(self, theta, phi, lam, ctl, tgt):
     """Apply cU3 gate from a specified control (ctl) to target (tgt) qubit
     with angle theta, phi, and lam.
-    A cU3-gate implements a theta, phi, and lam radian rotation of the qubit state vector about the
-    x, y, and z-axis, respectively of the Bloch sphere when the control qubit is in state |1>.
+    A cU3 gate implements a theta, phi, and lam radian rotation of the qubit state vector about the
+    x, y, and z axis, respectively of the Bloch sphere when the control qubit is in state |1>.
```

Comment:
Should be updated to match definition of U3 (or just say 'implements a U3(theta,phi,lam) on target if control...')

Reply:
done